src/commands/cmd_server.cc

```diff
@@ -309,6 +309,54 @@ class CommandDBSize : public Commander {
   }
 };
 
+class CommandKProfile : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    CommandParser parser(args, 1);
+    if (!parser.EatEqICase("memory")) {
+      return {Status::NotOK, "only supports MEMORY subcommand"};
+    }
+    if (parser.EatEqICase("enable")) {
+      enabled_ = true;
+    } else if (parser.EatEqICase("disable")) {
+      enabled_ = false;
+    } else if (parser.EatEqICase("dump")) {
+      if (!parser.Good()) return {Status::NotOK, errWrongNumOfArguments};
+      dump_dir_ = GET_OR_RET(parser.TakeStr());
+    } else {
+      return {Status::NotOK, "MEMORY subcommand must be one of ENABLE, DISABLE, DUMP"};
+    }
```

Comment:
Should we introduce a subcommand to check the current status? e.g. `kprofile memory enabled` -> true/false.

Reply:
It's nice to have. How about `kprofile memory status` -> `enabled|disabled`?

Reply:
The allocator name is now showing at the `info memory` section, and using the profiling command on an unsupported allocator will return an error for now. So I think it should be good to just return `enabled` or `disabled`.